_layouts/base.html.haml

```diff
@@ -30,6 +30,17 @@
     %link{:rel => "shortcut icon", :href => relative("/favicon.ico")}
 
   %body(data-spy="scroll" data-target=".bs-docs-sidebar")
+<!--Place GTM Code just below the opening body tag -->
+<script>dataLayer = [{'channel' : 'OptaPlanner', 'additional_tracking_code' : '#{site.googleAnalyticsId}'}];</script>
+<!-- Google Tag Manager -->
+<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-NJWS5L"
```

Comment:
Extracted 2 additional site variables (in case drools.org and jbpm.org want to copy paste these changes):
googleTagManagerId: GTM-NJWS5L
googleTagManagerChannel: OptaPlanner